docs/nodes/validators.md

```diff
@@ -67,19 +68,21 @@ When initializing nodes there are five parameters in the `config.toml` that may
 
 | Config Option            | Setting                    |
 | ------------------------ | -------------------------- |
+| mode                     | validator                  |
 | pex                      | false                      |
 | persistent-peers         | list of sentry nodes       |
 | private-peer-ids         | none                       |
 | unconditional-peer-ids   | optionally sentry node IDs |
 | addr-book-strict         | false                      |
 | double-sign-check-height | 10                         |
 
-The validator node should have `pex=false` so it does not gossip to the entire network. The persistent peers will be your sentry nodes. Private peers can be left empty as the validator is not trying to hide who it is communicating with. Setting unconditional peers is optional for a validator because they will not have a full address books.
+To run the node as validator must set `mode=validator`, The validator node should have `pex=false` so it does not gossip to the entire network. The persistent peers will be your sentry nodes. Private peers can be left empty as the validator is not trying to hide who it is communicating with. Setting unconditional peers is optional for a validator because they will not have a full address books.
```

Comment:
```suggestion
To run the node as validator ensure `mode=validator`. The validator node should have `pex=false` so it does not gossip to the entire network. The persistent peers will be your sentry nodes. Private peers can be left empty as the validator is not trying to hide who it is communicating with. Setting unconditional peers is optional for a validator because they will not have a full address books.
```